SPECS/glusterfs/glusterfs.spec

```diff
@@ -678,47 +973,77 @@ touch %{buildroot}%{_sharedstatedir}/glusterd/nfs/nfs-server.vol
 touch %{buildroot}%{_sharedstatedir}/glusterd/nfs/run/nfs.pid
 %endif
 
-## Install bash completion for cli
-install -p -m 0755 -D extras/command-completion/gluster.bash \
-    %{buildroot}%{_sysconfdir}/bash_completion.d/gluster
+find ./tests ./run-tests.sh -type f | cpio -pd %{buildroot}%{_prefix}/share/glusterfs
+
+%if 0%{!?mariner}
+# Mariner does not require a clean stage
+%clean
+rm -rf %{buildroot}
+%endif
```

Comment:
No need to comment, just remove.
```suggestion
```

Reply:
all the changes here are upstream changes.  my goal was to align to upstream except when the mariner variable is used to define behavior specific to mariner.  my preference would be to make the choice overt so the next upgrader can understand.

Reply:
modified per suggestion